src/backend/marsha/core/defaults.py

```diff
@@ -2,21 +2,39 @@
 from django.utils.translation import gettext_lazy as _
 
 
-PENDING, PROCESSING, ERROR, READY, IDLE, STARTING, STOPPED, RUNNING = (
+(
+    PENDING,
+    PROCESSING,
+    PROCESSING_LIVE_TO_VOD,
+    ERROR,
+    READY,
+    IDLE,
+    STARTING,
+    STOPPED,
+    RUNNING,
+    WAITING_LIVE_TO_VOD,
```

Comment:
can we make it a generic state for a video that is ready but that we don't want to show? hidden?
Hmm actually I think this is not an information of the state but another info no? Should it be another boolean field "is_visible_to_students"?

Reply:
there is already a props set in the serializer: `is_ready_to_show`. I think it's enough.

Reply:
I was talking about database fields. That it could be orthogonal to the start and serve to explicitly hide a video whatever it's state? WDYT?

Reply:
This is something that can be changed only in the admin ?

Reply:
not sure what you meant. In the case of a video that is HARVESTING you would present a button, "make it visible to students (disclaimer: if you don't we will destroy it in 7 days)"

Reply:
Fow now we don't have a column in the video table because to know if the video is visible to a student we compute the value depending of several parameters (`uploaded_on` value, `upload_state` value and `live_state` value). 
So if this database field is added, how does it work ? How do we change its value ?

Reply:
I would say the intructor can modify it from the dashboard?
And in the case of live videos that have been harvested and are in the grace period, a more obvious call to action can also be proposed?

Reply:
The call to action is only to change the `upload_state` in `READY` and we have a VOD. I think I don't understand what you want.

Reply:
as discussed IRL I was overlooking the fact that a live video can't be in the READY state until the instructor has clicked the button to confirm he/she wants it as a VOD. Otherwise we wouldn't be able to differentiate between an harvested video waiting for conversion to  VOD and a ready video that has been hidden... And only the former needs to be deleted...